hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -335,32 +335,87 @@ def mask_passwords_inplace(object_to_be_masked):
     Note that this method alters "object_to_be_masked".
     '''
     try:
-        mask_file  = __salt__['cp.cache_file']('salt://hubblestack_nebula_v2/mask.yaml')
-        if not mask_file:
-          return
-        mask = yaml.safe_load(open(mask_file))
+
+        mask = {}
+        topfile = 'salt://hubblestack_nebula_v2/top.mask'
```

Comment:
instead of writing file paths in code, these should be written in some other file that contains all constants. By looking at the path I feel there will, someday, be a **hubblestack_nebula_v3**, due to which changes in the function definition will be made. Hence increasing maintainability.